tests/console/hostname.pm

```diff
@@ -22,6 +22,7 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    assert_script_run "systemctl -q is-active network.service && systemctl restart network.service";
```

Comment:
I reuse HDD image generated by HA installation tests, I need to have different hostnames on different hosts. When you change hostname and if the machine use DHCP, it should send a new DHCP request after hostname change, and now hostname.pm for some reason doesn't do that.